pages/installation/os/debian.rst

```diff
@@ -2,62 +2,58 @@
 Debian installation
 *******************
 
-This guide describes the fastest way to install Graylog on Debian Linux 8 (Jessie). All links and packages are present at the time of writing but might need to be updated later on.
+This guide describes the fastest way to install Graylog on Debian Linux 9 (Stretch). All links and packages are present at the time of writing but might need to be updated later on.
 
 .. warning:: This setup should not be done on publicly exposed servers. This guide **does not cover** security settings!
 
 
 Prerequisites
 -------------
 
-Not all required dependencies are available in the standard repository, so we need to add `Debian Backports <https://backports.debian.org>`__ to the list of package sources::
-
-  $ sudo echo "deb http://ftp.debian.org/debian jessie-backports main" > /etc/apt/sources.list.d/backports.list
-  $ sudo apt-get update && sudo apt-get upgrade
-
 If you're starting from a minimal server setup, you will need to install these additional packages::
 
-  $ sudo apt-get install apt-transport-https openjdk-8-jre-headless uuid-runtime pwgen
+  $ sudo apt update && sudo apt upgrade
+  $ sudo apt install apt-transport-https openjdk-8-jre-headless uuid-runtime pwgen
 
 
 MongoDB
 -------
 
-The version of MongoDB included in Debian Jessie is recent enough to be used with Graylog 2.0.0 and higher::
+The version of MongoDB included in Debian Jessie is recent enough to be used with Graylog 2.3.x and higher::
 
-  $ sudo apt-get install mongodb-server
+  $ sudo apt install mongodb-server
 
 
 Elasticsearch
 -------------
 
-Graylog 2.0.0 and higher requires Elasticsearch 2.x, so we took the installation instructions from `the Elasticsearch installation guide <https://www.elastic.co/guide/en/elasticsearch/reference/2.3/setup-repositories.html#_apt>`__::
+Graylog 2.3.x can be used with Elasticsearch 5.x, please follow the installation instructions from `the Elasticsearch installation guide <https://www.elastic.co/guide/en/elasticsearch/reference/current/deb.html>`__::
```

Comment:
The link to the Elasticsearch documentation should use a specific version:

https://www.elastic.co/guide/en/elasticsearch/reference/5.4/deb.html